app/models/miq_worker/systemd_common.rb

```diff
@@ -0,0 +1,144 @@
+require "dbus/systemd"
```

Comment:
Is it possible to delay load this?

Reply:
Yeah I'll require right before opening up the `DBus::Systemd::Manager.new` connection